.azure-pipelines/ut-3x-pt.yml

```diff
@@ -11,6 +11,7 @@ pr:
       - neural_compressor/common
       - neural_compressor/torch
       - test/3x/torch
+      - test/3x/common
```

Comment:
I support the common test case not related to FWs, so when the test/3x/common changed, only ut-3x-pt test triggered.